scripts/wdio/wdio-runner.js

```diff
@@ -40,6 +40,10 @@ async function wdioRunner(options) {
 
       for (let factor = 0; factor < factors.length; factor += 1) {
         let envValues = `LOCALE=${locale} `;
+        if (theme) {
+          envValues += `THEME=${theme} `;
```

Comment:
Isn't this set above on line 34? https://github.com/cerner/terra-toolkit/pull/392/files#diff-d29f1ddc663577981f7d0700da2250edR34

Reply:
no, the test runner outputs `envValues`  on line [54/58](https://github.com/cerner/terra-toolkit/pull/392/files#diff-d29f1ddc663577981f7d0700da2250edR58) to indicate which values are being tested, per test run. we could rename this variable to `envTestRunnerOutput` or something similar? i agree, it's confusing

Reply:
meh I think this overall needs clean up  but better for a diff time 😄